src/components/Theme/src/Theme.vue

```diff
@@ -123,24 +123,49 @@ export default {
 			const MAX_THUMBNAIL_RADIUS = 8;
 
 			return {
+				// neutral colors
 				'--maker-color-neutral-0': colors['neutral-0'],
 				'--maker-color-neutral-10': colors['neutral-10'],
 				'--maker-color-neutral-20': colors['neutral-20'],
 				'--maker-color-neutral-80': colors['neutral-80'],
 				'--maker-color-neutral-90': colors['neutral-90'],
 				'--maker-color-neutral-100': colors['neutral-100'],
+
+				// general colors
 				'--maker-color-primary': colors.primary,
 				'--maker-color-background': colors.background,
 				'--maker-color-heading': colors.heading,
 				'--maker-color-body': colors.body,
 				'--maker-color-elevation': colors.elevation,
 				'--maker-color-overlay': colors.overlay,
+
```

Comment:
See above - I'm leery of adding contextual colors to the public Maker color variables.